src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/ChangeDirectionDueDateNotificationHandler.java

```diff
@@ -41,7 +42,11 @@ public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCa
         requireNonNull(callback, "callback must not be null");
 
         return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
-               && callback.getEvent() == Event.CHANGE_DIRECTION_DUE_DATE;
+               && callback.getEvent() == Event.CHANGE_DIRECTION_DUE_DATE
+               && (Arrays.asList(
+                    State.AWAITING_RESPONDENT_EVIDENCE,
+                    State.RESPONDENT_REVIEW
```

Comment:
@Lee365 I was thinking of having the DirectionParties check as well similar to what you have in case-api. That way to restrict it only for Respondent party than any other.

Reply:
Changed this to be in sync with case-api checks.